lib/ansible/plugins/callback/logdna.py

```diff
@@ -0,0 +1,204 @@
+# (c) 2018, Samir Musali <samir.musali@logdna.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: logdna
+    callback_type: aggregate
+    short_description: Sends playbook logs to LogDNA
+    description:
+      - This callback will report logs from playbook actions, tasks, and events to LogDNA (https://app.logdna.com)
+    version_added: 2.5.2
+    requirements:
+      - whitelisting in configuration
+    options:
+      conf_key:
+        required: True
+        description: LogDNA Ingestion Key
+        env:
+          - name: LOGDNA_INGESTION_KEY
+      plugin_ignore_errors:
+        required: False
+        description: Whether to ignore errors on failing or not
+        env:
+          - name: ANSIBLE_IGNORE_ERRORS
+        default: False
+      conf_hostname:
+        required: False
+        description: Alternative Host Name
```

Comment:
you should note it will default to the current host, but dont use the default field